src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1054,6 +1193,66 @@ public override BoundNode VisitIsPatternExpression(BoundIsPatternExpression node
             return node;
         }
 
+        public override void VisitPattern(BoundPattern pattern)
+        {
+            Visit(pattern);
+            base.VisitPattern(pattern);
```

Comment:
>base.VisitPattern(pattern); [](start = 12, length = 27)

Consider inlining the (simple) base implementation here. Otherwise it looks like both this and base are doing the same work.